misc/python/materialize/mzcompose/__init__.py

```diff
@@ -64,6 +64,7 @@
     "enable_columnation_lgalloc": "true",
     "enable_comment": "true",
     "enable_disk_cluster_replicas": "true",
+    "enable_eager_delta_joins": "false",
```

Comment:
Shouldn't we enable this in CI? CC @def-

Reply:
My understanding is that we plan to enable this in staging + production, so definitely!

Reply:
=> https://github.com/MaterializeInc/materialize/pull/24506

Reply:
Oh... thank you! I misunderstood what this flag was for.